app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
There is no need to put imports inside of `()` in case you leave them on a same line, replace this import with 
```suggestion
from app.errors import VaccineError, NotWearingMaskError
```